docs/langref/relay/expressions.rst

```diff
@@ -0,0 +1,162 @@
+===========
+Expressions
+===========
+
+The Relay IR is a pure, expression oriented language, with a dataflow fragment
+and structured control flow. Although Relay's representation is a tree, it is
+possible to view the dataflow fragments as graph for purposes of writing and
+expressing transformations.
+
+The below sections make an attempt to clearly split the dataflow
+fragment from the control fragment.
+
+====================
+Dataflow Expressions
+====================
+
+First we will cover the set of nodes which do not involve control flow,
+this fragment of the language is semantically equivalent to pure
+computation graphs without control flow.
+
+Constants
+~~~~~~~~~
+
+Relay programs can contain constant Tensor values. This node represents
+a constant tensor value (values are either Tensors, Products, or Closures in Relay).
+The constants are represented as :py:class:`~tvm.NDArray`, allowing us to utilize
+TVM operators for constant evaluation.
+
```

Comment:
Unnecessary capitalization in the above paragraph ("Tensor," "Products," "Closures")